CMakeLists.txt

```diff
@@ -132,18 +146,23 @@ endif()
 # Python module
 if(TRITON_BUILD_PYTHON_MODULE)
     message(STATUS "Adding Python module")
+    set(PYTHON_SRC_PATH ${CMAKE_CURRENT_SOURCE_DIR}/python/src)
```

Comment:
Some of these changes don't seem to be ROCm-specific. Could you help explain the rationale behind the changes?

Reply:
From the look of it, it's from upstream merge. In order to reduce confusion, please separate upstream merge into another PR. Keep ROCm-specific changes here.